apps/feedahelp/src/components/ui/features/deshiMenus/DeshiMenus.tsx

```diff
@@ -23,6 +25,15 @@ type DeshiMenuProps = {
 
 const DeshiMenus = () => {
   const [DeshiMenus, setDeshiMenus] = useState<DeshiMenuProps[]>([]);
+  const [modalOpen, setModalOpen] = useState(false);
+
+  const closeModal = (): void => {
+    setModalOpen(false);
+  };
+
+  const openModal = (): void => {
+    setModalOpen(true);
+  };
```

Comment:
what if we just pass the setModalOpen to the children??

Reply:
yeah, we can but just easy to understand. it will change when we implement redux its comming